client/src/includes/a11y-result.test.ts

```diff
@@ -55,3 +55,64 @@ describe('sortAxeViolations', () => {
     ]);
   });
 });
+
+describe('checkImageAltText', () => {
```

Comment:
Since those test cases are almost all identical, we should refactor as many as we could so they’re easier to review and update. As-is it’s very verbose so hard to tell exactly what pattern each test is for.

Try something like:

```js
describe.each`
  text                                                | result
  ${'Good alt text with words like GIFted and moTIF'} | ${true}
  ${'Bad alt text.png'}                               | ${false}
`('checkImageAltText', ({ text, result }) => {
  test(text, () => {
    const image = document.createElement('img');
    image.setAttribute('alt', text);
    expect(checkImageAltText(image)).toBe(result);
  });
});
```